src/assign.ts

```diff
@@ -187,6 +187,11 @@ export const selectReviewers = async (
       }
       debug(`Found random team member: ${randomTeamMember}.`)
 
+      if (randomTeamMember && randomTeamMember === author) {
+        debug(`'${randomTeamMember}' is the author '${randomTeamMember}'. Skipping.`)
+        continue
+      }
```

Comment:
We already assert that `randomTeamMember` isn't `undefined | null` in L:181 😄 

```suggestion
      if (randomTeamMember === author) {
        debug(`'${randomTeamMember}' is the author '${author}'. Skipping.`)
        continue
      }
```